### PR TITLE
18.6: blocks and blank line: add example with array

### PR DESCRIPTION
**Pull request by christophehurpeau**
```js
// bad
const arr = [
  function foo() {
  },
  function bar() {
  },
];
return arr;

// good
const arr = [
  function foo() {
  },

  function bar() {
  },
];

return arr;
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1415,6 +1415,26 @@
     };
 
     return obj;
+
+    // bad
+    const obj = [
```
Comment (justjake) [inline README.md]: maybe `arr` would be a better reference name here? I thought this was a typo for a moment -_-

Comment (ljharb) [inline README.md]: +1, obj means `{}` :-)

Comment (christophehurpeau) [inline README.md]: yes of course ! Sorry, changed it :)

**Comment by cyrilf** [comment]
:+1:

**Comment by luqin** [comment]
:+1:

**Comment by christophehurpeau** [comment]
PR for jscs: https://github.com/jscs-dev/node-jscs/pull/1546

**Pull request merged**